server/server.js

```diff
@@ -1,48 +1,48 @@
-const http = require('http');
-const app = require('./app');
+const http = require("http");
+const app = require("./app");
 
-const normalizePort = val => {
-  const port = parseInt(val, 10);
+const normalizePort = (val) => {
+	const port = parseInt(val, 10);
 
-  if (isNaN(port)) {
-    return val;
-  }
-  if (port >= 0) {
-    return port;
-  }
-  return false;
+	if (isNaN(port)) {
+		return val;
+	}
+	if (port >= 0) {
+		return port;
+	}
+	return false;
 };
-const port = normalizePort(process.env.PORT || '8080');
-app.set('port', port);
+const port = normalizePort(process.env.PORT || "3000");
```

Comment:
react app runs on port 3000 by default, this  might lead to error on dev env, something like `port is areal in use`, and this will lead to a confusion, its worth changing this to any port other than 3000, e.g 3001